### PR TITLE
Username Assistance: Changed Message

### PR DESCRIPTION
**Pull request by mjansenDatabay**
Mantis Issue: #22733

### CODE REVIEW

File: lang/ilias_de.lang [hunk 1]
```diff
@@ -7558,6 +7558,7 @@ pwassist#:#pwassist_invalid_username_or_email#:#Bitte korrigieren Sie die Werte
...
 pwassist#:#pwassist_mail_sent#:#ILIAS hat eine E-Mail an die Adresse %1$s versandt.\nWeitere Anweisungen finden Sie in dieser Mail.
+pwassist#:#pwassist_mail_sent_generic#:#ILIAS hat eine E-Mail an die hinterlegte Adresse versandt.\nWeitere Anweisungen finden Sie in dieser Mail.
```
Comment (klees) [inline lang/ilias_de.lang:7560]: Why do we duplicate the message?

File: Services/Init/classes/class.ilPasswordAssistanceGUI.php [hunk 2]
```diff
@@ -636,47 +636,14 @@ public function submitUsernameAssistanceForm()
...
-
-		// Retrieve a user object with matching user name and email address.
+		$email  = $form->getInput('email');
 		$logins = ilObjUser::_getUserIdsByEmail($email);
 
-		// No matching user object found?
-		// Show the password assistance form again, and display an error message.
-		if(!is_array($logins) || count($logins) < 1)
-		{
-			ilUtil::sendFailure(str_replace("\\n", '', $this->lng->txt('pwassist_invalid_email')));
```
Comment (klees) [inline Services/Init/classes/class.ilPasswordAssistanceGUI.php:649]: Could we remove `pwassist_invalid_email`?

**Review by klees — CHANGES_REQUESTED** [review]
LGTM, only minor change requests. Thx! Also: we have less code afterwards!

File: lang/ilias_de.lang (hunk 1 above)
Comment (mjansenDatabay) [reply lang/ilias_de.lang:7560]: Because the other message is still in use.

File: Services/Init/classes/class.ilPasswordAssistanceGUI.php (hunk 2 above)
Comment (mjansenDatabay) [reply Services/Init/classes/class.ilPasswordAssistanceGUI.php:649]: I removed this.

**Review by klees — APPROVED** [review]
LGTM, thx!

**Comment by alex40724** [comment]
+1

**Review by chfsx — APPROVED** [review]
LGTM, thanks!

**Comment by Amstutz** [comment]
TB: approves, merge plz.

**Pull request merged**